.travis.yml

```diff
@@ -63,6 +63,13 @@ deploy:
     on:
       branch: master
       condition: $BUILD_INSTALLER == yes && $TRAVIS_EVENT_TYPE == cron
+  - provider: pypi
+    user: "${PYPI_USER}"
+    password: "${PYPI_PASSWORD}"
+    distributions: "sdist bdist_wheel"
+    on:
+      tags: true
+      condition: $DEPLOY_PYPI == yes
```

Comment:
`repo: streamlink/streamlink`
https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

on every deploy provider would be nice,

currently `git push origin 0.12.1` on a **fork** with travis-ci will fail

https://travis-ci.org/back-to/streamlink/jobs/382711711#L2592

---

and all the `PyPI` code from https://github.com/streamlink/streamlink/blob/master/script/release.sh
can be removed

Reply:
Yeah, still need to update the release script - I'm hoping it can be removed soon :)

Reply:
I added the `repo` condition :)